modules/apps/callbacks/callbacks_test.go

```diff
@@ -290,26 +287,3 @@ func (s *CallbacksTestSuite) AssertHasExecutedExpectedCallbackWithFee(
 	}
 	s.AssertHasExecutedExpectedCallback(callbackType, isSuccessful)
 }
-
-// GetExpectedEvent returns the expected event for a callback.
-func GetExpectedEvent(
```

Comment:
I think this should be added back, looks like maybe it got removed accidentally?

Reply:
is it actually being used anywhere?

Reply:
Yea, I will add back. The usage was deleted :)